app/controllers/employees_controller.rb

```diff
@@ -1,48 +0,0 @@
-class EmployeesController < ApplicationController
-  def index
-    @employees=Employee.all
```

Comment:
you can also use Employee.select(:id, :name, :xyz)
using this you can pass required data only